src/app/pages/ParatimeDashboardPage/TestnetFaucet.tsx

```diff
@@ -8,6 +8,8 @@ import { SnapshotCard } from '../../components/Snapshots/SnapshotCard'
 import { COLORS } from '../../../styles/theme/colors'
 import { faucet } from '../../utils/externalLinks'
 import { Layer } from '../../../oasis-nexus/api'
+import { NativeTicker, Ticker } from '../../../types/ticker'
```

Comment:
Just remove this link warning. "'Ticker' is defined but never used"

Reply:
Fixed